

<launch>
	<arg name="im_topic" default="/car/camera/color/image_raw" />
	<arg name="out_dir" default="$(find mushr_visual_servo)/detections" />
	<arg name="box_topic" default="/image_detector/box" />
	<arg name="write_images" default="false" />

	<node pkg="mushr_visual_servo" type="image_detector.py" name="image_detector" output="screen">
		<param name="im_topic" value="$(arg im_topic)" />
		<param name="out_dir" value="$(arg out_dir)" />
		<param name="box_topic" value="$(arg box_topic)" />
		<param name="write_images" value="$(arg write_images)" />
	</node>

</launch>
